### PR TITLE
Installs Rspec and Shoulda Matcher gems

### PR DESCRIPTION
**Pull request by leiter007**
Background
Need to install RSpec and Shoulda matchers to run the Unit tests on the Legacy code challenge.

Solution
Adds the gems in the Gemfile and run bundle install, followed by ``` bundle exec rails generate rspec:install```. Configures the rspec file to require rails_helper instead of spec_helper.

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  # arbitrary gems may also be filtered via:
+  # config.filter_gems_from_backtrace("gem name")
+end
+
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails 
+  end
+end
```
Comment (faraznaeem) [inline spec/rails_helper.rb:24]: Please clean this file from comments

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  # the seed, which is printed after each run.
+  #     --seed 1234
+  config.order = :random
+
+  # Seed global randomization in this process using the `--seed` CLI option.
+  # Setting this allows you to use `--seed` to deterministically reproduce
+  # test failures related to randomization by passing the same `--seed` value
+  # as the one that triggered the failure.
+  Kernel.srand config.seed
+=end
```
Comment (faraznaeem) [inline spec/spec_helper.rb]: And this as well

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: spec/rails_helper.rb (hunk 1 above)
Comment (leiter007) [reply spec/rails_helper.rb:24]: Done

File: spec/spec_helper.rb (hunk 2 above)
Comment (leiter007) [reply spec/spec_helper.rb]: and done

**Comment by leiter007** [comment]
Deleted pry-byebug sinnce pry_rails already installed and more preferable for a Rails project.

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**